sui/rotate.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+
+# Environment Variables
+
+NAMESPACE="testnet"
+NAMESPACE2="testnet-amplifiers"
+WORKER="ampd-axelar-amplifier-worker-0"
+BASTION="axelar-core-bastion-bastion-6f5ddc97c5-jl45c"
+BASTION_CONTAINER="axelar-core-bastion"
+SERVICE_NAME="amplifier"
+MULTISIG_PROVER_ADDRESS="axelar1v8jrupu2rqpskwgtr69max0ajul92q8z5mdxd505m2hu3xc5jzcqm8zyc6"
+VOTING_VERIFIER_ADDRESS="axelar1sykyha8kzf35kc5hplqk76kdufntjn6w45ntwlevwxp74dqr3rvsq7fazh"
+SERVICE_REGISTRY="axelar1rpj2jjrv3vpugx9ake9kgk3s2kgwt0y60wtkmcgfml5m3et0mrls6nct9m"
+
+# On mainnet there will be a rotation delay, so we can only run one iteration at a time
+ITERATIONS=1
+
+print_section() {
+    echo ""
+    echo ""
+    echo "============================== $1 =============================="
+}
+
+print_success() {
+    echo ""
+    echo "[✔] $1"
+}
+
+print_info() {
+    echo ""
+    echo "[INFO]: $1"
+}
+
+# Functions
+set_namespace() {
+    print_section "Setting Namespace"
+    kubectl config set-context --current --namespace="$NAMESPACE"
+    print_success "Namespace set to $NAMESPACE"
+}
+
+query_current_verifier_set() {
+    kubectl exec -it "$BASTION" -c "$BASTION_CONTAINER" -- bash -c "axelard q wasm contract-state smart $1 '\"current_verifier_set\"'"
+}
+
+query_next_verifier_set() {
+    kubectl exec -it "$BASTION" -c "$BASTION_CONTAINER" -- bash -c "axelard q wasm contract-state smart $1 '\"next_verifier_set\"' -o json"
+}
+
+get_worker_address() {
+    kubectl exec -it "$WORKER" -n "$NAMESPACE2" -c ampd-verifier-address -- bash -c "cat verifier-address"
+}
+
+query_active_verifiers() {
+    kubectl exec -it "$BASTION" -c "$BASTION_CONTAINER" -- bash -c \
+        "axelard q wasm contract-state smart $SERVICE_REGISTRY '{\"active_verifiers\":{\"service_name\": \"$SERVICE_NAME\",\"chain_name\":\"sui\"}}'"
+}
+
+register_or_deregister() {
```

Comment:
Shouldn't be part of this script. The caller can do this before/after calling the rotation script